### ISSUE TITLE
Redesign visitor log

### ISSUE THREAD

**Issue by tsteur** [piwik/piwik#7909]
Follow up for #7812 

Current problems are for example:
* Text is often cut off as each column is not that wide
* There is sometimes a lot of whitespace eg when there are many actions
* It doesn't look really nice in general. The current design is probably many years old
* While fixing this we can also fix the HTML/CSS of the visitor log. Eg because of this titles/urls are sometimes cut off too early
* ~~Maybe we can always use a 2 column layout?~~

**Comment by mattab** [comment]
The Visitor Log is one of the most popular features in Piwik. We think the feature is really powerful, but the look and feel can be improved. Ultimately we will work on even more awesome real time reporting in Piwik https://github.com/piwik/piwik/issues/2439 but in the meantime, we want to make Visitor log look nicer and more professional.

**Comment by mattab** [comment]
We also need to re-design the Live widget. If it is not included in this issue, when we close this issue, let's create one for "Redesign Live! real time widget" :+1:

**Comment by tsteur** [comment]
Many improvements were made in #7090

**Comment by Amenel** [comment]
Hi, I've just updated to 2.14.0 and the first change that struck me is that no matter how wide the screen is, each headline (the grey area) of entries in the Live widget now has the flag (and other icons like the browser and OS) showing up on the next line. Things were different in the previous version(s) (see http://piwik.org/docs/real-time/) as all info was on the same line and the user's browser handled the line wrapping.

Using Firebug, I see that there is a <br/> tag before the <img> tag that contains the country flag. So the question is "was this intended as a result of this UI redesign?" or is it an oversight?

Anyway, I think it was not the best choice if a choice it was indeed; these days, most screens are wide and vertical screen estate is much needed. I checked on a website of mine still running a previous version and it has date+time, country, browser, OS, profile icon, hyphen, and IP address on a single line. It felt right to me all these years. Simply put, the current state of affairs feels awkward to me (e.g. profile icon being separated from IP address).

It would be better to replace that <br/> with a <span> styled with a class. Then, any knowledgeable enough user of Piwik can have the line break overridden in a user.css file. With HTML markup, we have no choice but to accept this suboptimal layout. Moreover, if someone were to decide to create a theme, this would limit them, wouldn't it?

Maybe that these icons (flag, browser and OS) should also be grouped in a specific span or div.

For now, I've added .datetime > br { display: none;} to my userContent.css, hoping it'll work.
